apps/site/next-data/releaseData.ts

```diff
@@ -17,10 +17,14 @@ const getReleaseData = (): Promise<Array<NodeRelease>> => {
     );
   }
 
+  const fetchURL = `${NEXT_DATA_URL}release-data`;
+
   // When we're on RSC with Server capabilities we prefer using Next.js Data Fetching
   // as this will load cached data from the server instead of generating data on the fly
   // this is extremely useful for ISR and SSG as it will not generate this data on every request
-  return fetch(`${NEXT_DATA_URL}release-data`).then(r => r.json());
+  return fetch(fetchURL, { cache: 'force-cache' })
+    .then(response => response.text())
+    .then(JSON.parse);
```

Comment:
why changing this. r.json should still work ?

Reply:
The reason why this was done, was to not lose error context. If we simply keep response.json() Next.js will strip away which request is actually failing.

Even more as response.text() allows us to at least resolve the response even if it eventually fails on parsing the JSON.